readme.md

```diff
@@ -18,12 +18,14 @@ const cliBoxes = require('cli-boxes');
 console.log(cliBoxes.single);
 /*
 {
-    topLeft: '┌',
-    topRight: '┐',
-    bottomRight: '┘',
-    bottomLeft: '└',
-    vertical: '│',
-    horizontal: '─'
+	topLeft: '┌',
+	topRight: '┐',
+	bottomRight: '┘',
+	bottomLeft: '└',
+	verticalLeft: '│',
+	verticalRight: '│',
+	horizontalTop: '─',
+	horizontalBottom: '─'
```

Comment:
Needs to be updated.

Reply:
Good catch. Thought I did find and replace on all files.